CMakeLists.txt

```diff
@@ -52,56 +59,196 @@ if (NOT HAVE_CXX_ATOMICS_WITHOUT_LIB
+            endif ()
+        endif ()
+        if (NOT EXISTS "${gnutls_SOURCE_DIR}/include/gnutls")
+            file(MAKE_DIRECTORY "${gnutls_SOURCE_DIR}/include")
+            file(RENAME "${gnutls_SOURCE_DIR}/lib/includes" "${gnutls_SOURCE_DIR}/include/gnutls")
+        endif ()
+        set(GNUTLS_INCLUDE_DIR "${gnutls_SOURCE_DIR}/include")
+        if (BUILD_SHARED_LIBS)
+            set(GNUTLS_LIBRARY ${gnutls_SOURCE_DIR}/bin/libgnutls-30.dll)
+            link_directories(${gnutls_SOURCE_DIR}/bin)
+        else ()
+            set(GNUTLS_LIBRARY ${gnutls_SOURCE_DIR}/lib/libgnutls.dll.a)
+            link_directories(${gnutls_SOURCE_DIR}/lib)
+        endif ()
+        file(COPY
+            "${gnutls_SOURCE_DIR}/bin/libgnutls-30.dll"
+            "${gnutls_SOURCE_DIR}/bin/libidn2-0.dll"
+            "${gnutls_SOURCE_DIR}/bin/libnettle-8.dll"
+            "${gnutls_SOURCE_DIR}/bin/libssp-0.dll"
+        DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/bin")
+        
+        find_package(GnuTLS REQUIRED)
+        include_directories(${GNUTLS_INCLUDE_DIR})
+        
+        # C99 required for cJSON to handle 'long long' declarations on x64 systems
+        target_compile_options(cjson PRIVATE --std=c99 -Wno-float-conversion) 
+        target_compile_options(cJSON_test PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(parse_examples PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(parse_number PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(unity PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(parse_hex4 PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(parse_string PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(parse_array PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(parse_object PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(parse_value PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(parse_with_opts PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(print_string PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(print_number PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(print_array PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(print_object PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(print_value PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(compare_tests PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(cjson_add PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(misc_tests PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(minify_tests PRIVATE --std=c99 -Wno-float-conversion)
+        target_compile_options(readme_examples PRIVATE --std=c99 -Wno-float-conversion)
```

Comment:
should not be needed when no using fetchcontent

Reply:
Yes, as I will check on MSYS2, likely we will remove this